torch/_inductor/fx_passes/pad_mm.py

```diff
@@ -389,13 +456,13 @@ def should_pad_mm_bf16(dtype: torch.dtype, M: int, N: int, K: int) -> bool:
     return False
 
 
-def should_pad_bench(*args: Any, **kwargs: Any) -> bool:
+def should_pad(*args: Any, **kwargs: Any) -> bool:
```

Comment:
i worry this might tempt someone later to only call should_pad without can_pad. i like the distinction, but how about we carry it through: i.e. these are still is_padding_beneficial functions, and should_pad functions call both can_pad and is_padding beneficial?